ui/calendar.js

```diff
@@ -61,26 +61,27 @@ return $.widget( "ui.calendar", {
 		this.labels = Globalize.translate( "datepicker" );
 		this.buttonClickContext = this.element[ 0 ];
 
-		this.date = $.date( this.options.value, this.options.dateFormat ).select();
-		this.date.eachDay = this.options.eachDay;
+		this.date = $.date( this.options.value, this.options.dateFormat );
+		this.viewDate = this.date.clone();
+		this.viewDate.eachDay = this.options.eachDay;
 
 		this._on( this.element, {
 			"click .ui-calendar-prev": function( event ) {
 				event.preventDefault();
 				this.date.adjust( "M", -this.options.numberOfMonths );
-				this.refresh();
+				this._refresh();
```

Comment:
No, we want to change the current view as well as the working date. Don't confuse this.date with the currently selected date (which is set here: https://github.com/fnagel/jquery-ui/blob/datepicker-fix-jumping/ui/calendar.js#L374-L376).